client/src/components/organisms/Header/Header.test.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import Header from './Header';
+
+describe('<Header />', () => {
+  const wrapper = mount(<Header />);
+  describe('Logo', () => {
+    const logoWrapper = wrapper.find('.navbar-brand');
+    it('should contain exactly one logo', () => {
+      expect(logoWrapper.length).toBe(1);
+    });
+  });
+  describe('Buttons', () => {
+    const buttonsWrapper = wrapper.find('.btn');
+    it('should contain 2 buttons', () => {
+      expect(buttonsWrapper.length).toBe(2);
+    });
+    it('should contain 1 "Create Timeline" button', () => {
+      expect(buttonsWrapper.contains('Create Timeline')).toBe(true);
```

Comment:
not a good way to test for the presence of button. 
Someonce can change the text anytime. 
Try using data-id etc attribute